plugin/init.lua

```diff
@@ -141,41 +156,53 @@ end
 --- @param data string
 --- @return string
 local function sanitize_json(data)
+	wezterm.emit("resurrect.sanitize_json.start", data)
 	data = data:gsub("[\x00-\x1F\x7F]", function(c)
+		local byte = string.byte(c)
 		if is_windows then
-			local byte = string.byte(c)
 			if byte == 0x0A then
 				return "\n"
 			elseif byte == 0x0D then
 				return "\r"
 			elseif byte == 0x09 then
 				return "\t"
 			else
-				return string.format("\\u%04X", string.byte(c))
+				wezterm.log_info("Invalid control character: " .. byte)
+				return string.format("\\u%04X", byte)
 			end
 		else
-			return string.format("\\u00%02X", string.byte(c))
+			wezterm.log_info("Invalid control character: " .. byte)
```

Comment:
Same as comment on line 170

Reply:
Same as above. Useful for debugging but not necessary.

Reply:
Removed